x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/data_table/custom_timeline_data_grid_body.tsx

```diff
@@ -204,13 +213,16 @@ const CustomDataGridSingleRow = memo(function CustomDataGridSingleRow(
     eventId = '',
     onToggleShowNotes,
     refetch,
+    rowHeight = 0,
   } = props;
   const dispatch = useDispatch();
   const { canShowRowRenderer } = useStatefulRowRenderer({
     data: rowData.ecs,
     rowRenderers: enabledRowRenderers,
   });
 
+  let cssRowHeight: string = 'auto';
+  if (rowHeight >= 0) cssRowHeight = `${DEFAULT_UDT_ROW_HEIGHT + 16 * rowHeight}px`;
```

Comment:
would make sense to add explanation of this calculation. Why adding 16? Is it additional Unit that added for each row height increments. Might also make sense to make it a constant.

Reply:
absolutely agree with @logeekal. We should have a comment here for this whole calculation, or we (at least I) won't remember in 2 weeks 🤣